src/openrct2/interface/Window.cpp

```diff
@@ -2167,3 +2167,18 @@ void WindowBase::ResizeFrameWithPage()
     widgets[3].right = width - 1;
     widgets[3].bottom = height - 1;
 }
+
+void WindowBase::ResizeDropdown(WidgetIndex widgetIndex, const ScreenCoordsXY& origin, const ScreenSize& size)
+{
+    auto right = origin.x + size.width - 1;
+    auto bottom = origin.y + size.height - 1;
+    widgets[widgetIndex].left = origin.x;
+    widgets[widgetIndex].top = origin.y;
+    widgets[widgetIndex].right = right;
+    widgets[widgetIndex].bottom = bottom;
+
+    widgets[widgetIndex + 1].left = right - size.height + 1;
```

Comment:
Why is the height subtracted here? I'm guessing it's to have the button be square regardless of the widget height?

Reply:
Yep.

Reply:
It'll be nice to comment that, because without an explanation, subtracting height from width looks wrong.

Reply:
Added the comment